starknet/compiler.go

```diff
@@ -1,5 +1,7 @@
 package starknet
 
+// why it's in separate package (not in VM), is it because of cyclic dependencies ?
+
```

Comment:
I believe it is mainly because I found it cleaner to have it here instead of VM.